### ISSUE TITLE
Make empty links buttons

### ISSUE THREAD

**Issue by beefoo** [NYPL/design#8]
E.g.

```
<a href="#" title="Show only resources with name starting with A">A</a>
```

For screen reader compatibility, should be

```
<button title="Show only resources with name starting with A">A</button>
```

or

```
<a href="#" role="button" title="Show only resources with name starting with A">A</a>
```

It's also unclear what the best practices are for `title` attributes for buttons and links since the text inside the tag will be read initially

**Comment by mgiraldo** [comment]
yeah. that question question was raised in the docs.

same would apply for “Search results summary” links to remove a facet.

**Comment by mgiraldo** [comment]
moved to https://github.com/NYPL/design-toolkit/issues/8
